solidity/contracts/liquidity-protection/LiquidityProtectionSettings.sol

```diff
@@ -121,6 +134,15 @@ contract LiquidityProtectionSettings is ILiquidityProtectionSettings, AccessCont
      */
     event AverageRateMaxDeviationUpdated(uint32 _prevAverageRateMaxDeviation, uint32 _newAverageRateMaxDeviation);
 
+    /**
+     * @dev triggered when adding liquidity is disabled or enabled for a given reserve on a given pool
+     *
+     * @param _poolAnchor   pool anchor
+     * @param _reserveToken reserve token
+     * @param _state        true if disabled, false otherwise
```

Comment:
_state -> _disabled (I know it might look redundant but "state" isn't self explanatory).

Reply:
Fixed.